frontend/src/components/Layout.tsx

```
@@ -192,10 +196,13 @@ export default function Layout() {
             <Cpu className="w-4 h-4" />
             <button
               onClick={() => setShowWhatsNew(true)}
-              className="hover:text-gray-600 dark:hover:text-slate-300 dark:text-slate-300 transition-colors text-left"
+              className="hover:text-gray-600 dark:hover:text-slate-300 dark:text-slate-300 transition-colors text-left relative"
               aria-label="Show changelog"
             >
               Satellite Processor {versionInfo.display}
+              {hasNewVersion && (
+                <span className="absolute -top-1 -right-3 w-2.5 h-2.5 bg-primary rounded-full animate-pulse" aria-label="New version available" />
```

Biome (2.3.14)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)